src/content/developers/tutorials/send_token_etherjs.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Sending Tokens Using ethers.js
+description: Beginner friendly guide to sending tokens using ethers.js.
+author: Kim YongJun
+tags: ["ETHERS.JS", "ERC-20", "TOKENS"]
+skill: beginner
+lang: en
+sidebar: true
+published: 2021-04-06
+source:
+sourceUrl: 
+---
+
+
+
+## send Token Using ethers.js(5.0) {#send-token}
+
+### In This Tutorial You'll Learn How To {#you-learn-about}
+* Include ethers.js
+* Transfer token
+* Set gas price according to the network traffic situation
+
+### Prerequisite {#prerequisite}
+1. Include ethers.js(5.0)
+
+ES6 in the Browser
+```
+<script type="module">
+    import { ethers } from "https://cdn.ethers.io/lib/ethers-5.0.esm.min.js";
+    // Your code here...
+</script>
+```
+
+ES3(UMD) in the Browser
+```
+<script src="https://cdn.ethers.io/lib/ethers-5.0.umd.min.js"
+        type="application/javascript"></script>
+```
+### Parameter {#param}
+1. contract_address - Token contract address(contract address is needed when the token you want to transfer is not ether)
+2. send_token_amount - The amount you want to send to the receiver
+3. to_address - The receiver's address
+4. send_account - The sender's address
+5. private_key - Private key of the sender to sign the transaction and actually transfer the tokens
+
+## Notice {#notice}
+signTransaction(tx) is removed because sendTransaction() does it internally.
```

Comment:
Maybe condense this to one line within Parameters instead of it's own section:

**Notice**: `signTransaction(tx)` is removed because `sendTransaction()` does it internally.